src/scripts/migrations/__tests__/createBackendUsers.js

```diff
@@ -3,6 +3,7 @@ import client from 'util/client';
 import CreateBackendUsers from '../createBackendUsers';
 import fixtures from '../__fixtures__/createBackendUsers';
 import { sortBy } from 'lodash';
+jest.setTimeout(50000);
```

Comment:
haha, good catch.
Actually, I think this might only be needed for test to pass locally.  It doesn't seem to be timeout as easily on travis.
I am actually thinking maybe we can make timeout value an environment variable to be set in .env and move this to setup since this is pretty much environment preference dependent (I don't know about everyone else, but they run way slower on my laptop than on travis :p)

Reply:
I think it's OK that we increase timeout directly.

Increasing timeout does not impact time spent on test, but can save precious developing energy on problems that we can really solve.